internal/k8s/k8s.go

```diff
@@ -354,7 +354,7 @@ func (c *Client) CreateMlSecret(namespace, controllerDeploymentName, secretName
 		},
 		metav1.CreateOptions{})
 	if err == nil {
-		level.Info(c.logger).Log("op", "CreateMlSecret", "msg", "secret succesfully created")
+		level.Info(c.logger).Log("op", "CreateMlSecret", "msg", "secret successfully created")
```

Comment:
How comes this is here? I thought I merged it

Reply:
That's a good question. Cleaned this up and looks like it's gone.